irispy/iris_tools.py

```diff
@@ -96,7 +114,7 @@ def get_iris_response(pre_launch=False, response_file=None, response_version=Non
         if not(os.path.isfile(response_file)):
             raise KeyError("Not a valid file path")
 
-    
+
```

Comment:
Delete this empt line.  No need to two empty lines.

Reply:
I don't know why a empty line appears here !?

Deleted